### PR TITLE
Add docs template for spec issues

### PR DESCRIPTION
**Pull request by BillWagner**
Once this is merged I'll add the config to direct customer issues to this template.

The "View file" tab on changes will give a reasonable facsimile of the experience a user will see when they are routed to this form. The fields at the bottom will be filled it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/docs-feedback.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Learn feedback control.
+description: |
+  ⛔ This template is hooked into the feedback control on the bottom of every page on the learn.microsoft.com site. It automatically fills in several fields for you. Don't use for other purposes. ⛔
+assignees:
+  - BillWagner
```
Comment (BillWagner) [inline .github/ISSUE_TEMPLATE/docs-feedback.yml:5]: Who else should I add here?

I think I can add a team, as long as the org isn't needed (meaning its a team in the dotnet org). The GitHub docs are unclear.

**Comment by BillWagner** [comment]
The labels will be specified in the PR that updates the docs config. We've learned that those fields aren't obeyed when a URL routes directly to the template.

File: .github/ISSUE_TEMPLATE/docs-feedback.yml (hunk 1 above)
Comment (jcouv) [reply .github/ISSUE_TEMPLATE/docs-feedback.yml:5]: I'd rather not assign a team. It's preferable to have a single owner for each issue.
If you're okay routing/triaging incoming issues by default, that'd be great, but we can discuss some options if the load is too great.

**Review by jcouv — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/docs-feedback.yml (hunk 1 above)
Comment (BillWagner) [reply .github/ISSUE_TEMPLATE/docs-feedback.yml:5]: Triaging these hasn't been a major time sink in the past. I don't expect that to change, but I'll let everyone know if it is.

File: .github/ISSUE_TEMPLATE/docs-feedback.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+body:
```
Comment (333fred) [inline .github/ISSUE_TEMPLATE/docs-feedback.yml:6]: Can we add a `labels` section and apply some label like `speclet issue` or something like that?

Comment (333fred) [reply .github/ISSUE_TEMPLATE/docs-feedback.yml:6]: We'll need to make the label as well.

Comment (BillWagner) [reply .github/ISSUE_TEMPLATE/docs-feedback.yml:6]: The labels are added as a config in the docs repo. What we found during initial testing is that when the URL targets a specific issue template, some of the items in the template aren't honored (title and labels). So, we add add those in the docfx.json file in the docs repo, and the labels get passed on the URL.

I was thinking of adding the "not triaged" label, and a new "area-speclet" label.

Comment (333fred) [reply .github/ISSUE_TEMPLATE/docs-feedback.yml:6]: > The labels are added as a config in the docs repo.

The problem, though, is that this template will show up when people create a new issue from this repo as well. If they do, the right labels won't be applied.

Comment (BillWagner) [reply .github/ISSUE_TEMPLATE/docs-feedback.yml:6]: Good point. I'll add them in both.

**Comment by BillWagner** [comment]
I added an "Area-Speclets" label to match the other "Area-*" labels.

I updated the template to apply that label, and the "untriaged" label.

**Review by 333fred — APPROVED** [review]

**Pull request merged**